prboom2/src/hu_lib.c

```diff
@@ -145,6 +145,7 @@ void HUlib_drawTextLine
   int     i;
   int     w;
   int     x;
+  int     xbase;
```

Comment:
The question is whether there is any text that relies on the old behaviour. It's not exactly a bug, but that the old text start x was treated as an indentation for the first line, like in a paragraph.

Reply:
I couldn't find anything relying on that behaviour, but I was also concerned about how this change interacts with negative x values, which cause right justification. That is, it would not be a complete fix since the text across lines would not be properly justified.

But I couldn't find anything actually using that, so I just cleaned it up entirely.

https://github.com/kraflab/dsda-doom/commit/09f2b33de8d6f9f41c3d8e4f3b46a1f7234a5d06
https://github.com/kraflab/dsda-doom/commit/214f6e534b43b6eef12e15fff20038c865b4cb56

You may want to rebase and just add the grid argument commit on top 😺